### DISCUSSION TITLE
NVEnc section where to enable INFO option?

### DISCUSSION THREAD

**Discussion by Victory61** [staxrip/staxrip#874] [Q&A] [answered]
Good morning.
In the NVEnc section I can't find the INFO option as in the x265> Bitsream section.
Some idea?
Thanks

[img]https://i.postimg.cc/Hx3WP8hh/Screenshot-001.png[/img]

**Comment by Dendraspis** [comment, marked as answer]
NVEncC sadly does not offer this option.

**Reply by Victory61** [reply]
Thanks.
